README.md

```diff
@@ -90,10 +118,24 @@ def mystery5(s)
 end
 ```
 
-- What is mystery5("hi")?
-- What is mystery5("")?
-- What is mystery5("Hi, there!")?
+- What is mystery5("hi")?  **
+- What is mystery5("")? ""  (ie, empty string)
+- What is mystery5("Hi, there!")? **********
+
 - _Added Fun: How could we make only alphabetic characters to be changed to stars?_
+  def mystery5C(s)
+  alpha_all = ("A".."Z").to_a.concat ("a".."z").to_a
+  if s.length == 0
+    return ""
+  else
+    if alpha_all.include?(s[0])
+      return "*" + mystery5C(s[1..-1])
+    else
+      return mystery5C(s[1..-1])
+    end
+  end
+end
+
```

Comment:
This looks like it works. I did post another solution to consider.

One thing is to figure out if you think I intended mystery5("Hi, there!") to produce "*******" or "**, *****!" 

I intended the latter, but didn't specify so you+others assumed and wrote solutions for the former.